tests/phpunit/tests/media.php

```diff
@@ -3548,6 +3548,8 @@ public function data_attachment_permalinks_based_on_parent_status() {
 
 	/**
 	 * @ticket 53675
+	 * @ticket 56930
+	 * @covers ::wp_get_loading_attr_default
 	 * @dataProvider data_wp_get_loading_attr_default
```

Comment:
```suggestion
	 * @ticket 53675
	 * @ticket 56930
	 *
	 * @covers ::wp_get_loading_attr_default
	 *
	 * @dataProvider data_wp_get_loading_attr_default
```

Per [56070](https://core.trac.wordpress.org/ticket/56070) proposal, add one line break for annotation. need to apply similar changes in other tests.